src/app/shared/directives/highlight-status.directive.ts

```diff
@@ -0,0 +1,24 @@
+import { Directive, ElementRef, Input, OnChanges, Renderer2 } from '@angular/core';
+import * as moment from 'moment';
+
+@Directive({
+  selector: '[appHighlightStatus]'
+})
+export class HighlightStatusDirective implements OnChanges {
+
+  @Input('appHighlightStatus') date: Date;
+
+  constructor(private renderer: Renderer2, private hostElement: ElementRef) { }
+
+  ngOnChanges() {
+    const now = moment();
+    const date = moment(this.date);
+    const difference = moment.duration(now.diff(date));
+
+    if ( date > now) {
```

Comment:
consider adding linting on precommit git hook, to avoid formatting issues

Reply:
done